tests/unit/test_call_endpoint.py

```diff
@@ -0,0 +1,216 @@
+                        "endpoint_id": "7a489a3a-0152-4b01-b1cc-c10d7f84f0bc",
+                        "endpoint_java_id": "7a489a3a-0152-4b01-b1cc-c10d7f84f0bc",
+                        "endpoint_platform": "windows",
+                        "endpoint_type": "computer",
+                        "event_service_id": "69086f43-d619-4d03-a110-9a8cee3436f7",
+                        "inserted_at": 1554135257294,
+                        "source_info": {
+                            "ip": "10.1.39.32"
+                        },
+                        "user_match_id": "5ca22ac8e5a7400deb1ab0bd"
+                    },
+                    "type": "Event::Endpoint::NotProtected",
+                    "location": "Lightning-oxidtdinku",
+                    "description": "Failed to protect computer: Lightning-oxidtdinku",
+                    "source": "Lightning-g7n7wdv611\\Lightning"
+                },
+                {
+                    "severity": "high",
+                    "when": "2019-04-01T15:11:41.000Z",
+                    "threat": "Eicar-AV-Test",
+                    "event_service_event_id": "30acdaed-5387-4a39-b7cd-d7b9ac2a8c0f",
+                    "id": "30acdaed-5387-4a39-b7cd-d7b9ac2a8c0f",
+                    "info": None,
+                    "created_at": "2019-04-01T15:11:41.253Z",
+                    "customer_id": "816f36ee-dd2e-4ccd-bb12-cea766c28ade",
+                    "threat_cleanable": False,
+                    "data": {
+                        "created_at": 1554131501192,
+                        "endpoint_id": "c80c2a87-42f2-49b2-bab7-5031b69cd83e",
+                        "endpoint_java_id": "c80c2a87-42f2-49b2-bab7-5031b69cd83e",
+                        "endpoint_platform": "windows",
+                        "endpoint_type": "server",
+                        "event_service_id": "30acdaed-5387-4a39-b7cd-d7b9ac2a8c0f",
+                        "inserted_at": 1554131501192,
+                        "source_info": {
+                            "ip": "10.1.39.32"
+                        },
+                        "threat_id": "5ca22a2c352ea40df121ea9c",
+                        "user_match_id": "5ca22a0de5a7400deb1ab0bb"
+                    },
+                    "type": "Event::Endpoint::Threat::CleanupFailed",
+                    "location": "mock_Mercury_1",
+                    "description": "Manual cleanup required: 'Eicar-AV-Test' at 'C:\\Program Files (x86)\\Trojan Horse\\bin\\eicar.com'",
+                    "source": "n/a"
+                }
+            ]
+        }
+
+        alerts = []
+        # Run
+        try:
+            with mock.patch('siem.request_url') as mock_request_url:
+                mock_request_url.return_value = json.dumps(mock_alert_response)
+                for a in siem.call_endpoint(mock.Mock(), siem.ALERTS_V1, False,
+                                            False, 'fake_state_file', mock_config_token):
+                    alerts.append(a)
```

Comment:
This is a bit hard to read with the line break. Perhaps create a variable that stores the results from `call_endpoint` and then use that in the for-loop's definition. This isn't a blocker either.

Reply:
Done